pkg/awsutils/awsutils.go

```diff
@@ -228,17 +228,13 @@ func prometheusRegister() {
 //StringSet is a set of strings
 type StringSet struct {
 	sync.RWMutex
-	data 	sets.String
+	data sets.String
 }
 
-func (ss *StringSet) AWSStrings() []*string {
+func (ss *StringSet) SortedList() []string {
 	ss.RLock()
 	defer ss.RUnlock()
-	var dataSlice []*string
-	for key, _ := range ss.data {
-		dataSlice = append(dataSlice, aws.String(key))
-	}
-	return dataSlice
+	return ss.data.List()
```

Comment:
Can we leave a comment here saying that sets.String.List() returns a sorted list?
(I had to go look up the docs for that sets.String to see if this line was reasonable or not)